cmd/trace-agent/config/config_linux.go

```diff
@@ -0,0 +1,42 @@
+//go:build linux
+// +build linux
+
+package config
+
+import (
+	"errors"
+
+	"github.com/DataDog/datadog-agent/pkg/util/cgroups"
+	"github.com/DataDog/datadog-agent/pkg/util/pointer"
+	systemutils "github.com/DataDog/datadog-agent/pkg/util/system"
+)
+
+func getCgroupCPULimit() (float64, error) {
+	reader, err := cgroups.NewSelfReader("/proc", true)
```

Comment:
You cannot pass `true` as second parameter. Cgroups limit can also be set through `systemd` (host install)